javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,32 @@
+//For Peter
+const Pwide = 8;
+const Pdeep = 10;
+const Phigh = 10;
+let PvolumeInMeters;
+PvolumeInMeters = Pwide * Pdeep * Phigh;
+const PgardenSizeInM2 = 100;
+const PhouseCosts = 2500000;
+let PhousePrice;
+PhousePrice = PvolumeInMeters * 2.5 * 1000 + PgardenSizeInM2 * 300;
+if (PhouseCosts => PhousePrice) {
```

Comment:
There's a syntax in javascript called "fat arrow" that uses this character.
What you want to use here is `>=`.